plugins/linux-capture/pipewire-capture.c

```diff
@@ -167,9 +169,12 @@ void pipewire_capture_load(void)
 		obs_register_source(&pipewire_window_capture_info);
 
 	pw_init(NULL, NULL);
+	inited = true;
 }
 
 void pipewire_capture_unload(void)
 {
-	pw_deinit();
+	if (inited) {
+		pw_deinit();
+	}
```

Comment:
It's been a while since i perused the coding guidelines as I generally depend on clang-format. But it looks like the recommendation is no braces for single line ifs.

Reply:
As I understand it, Jim's not super picky about this, and personally leans towards always using braces to err on the side of caution.  I've stopped enforcing no braces for single line `if` statements in my reviews as a result.  Either way is fine for now.